DESCRIPTION

```diff
@@ -3,15 +3,25 @@ Type: Package
 Title: ADaM in R Asset Library - Dev Tools
 Version: 0.0.1
 Authors@R: c(
-    person("Open", "Source", email = "admiralext@pharmaverse.com", role = c("aut", "cre")))
-Description: A toolbox for programming Clinical Data Standards Interchange Consortium
-    (CDISC) compliant Analysis Data Model (ADaM) datasets in R. ADaM datasets are a
-    mandatory part of any New Drug or Biologics License Application submitted to the
-    United States Food and Drug Administration (FDA). Analysis derivations are
-    implemented in accordance with the "Analysis Data Model Implementation Guide"
-    (CDISC Analysis Data Model Team, 2021, <https://www.cdisc.org/standards/foundational/adam/adamig-v1-3-release-package>).
+    person("Ben", "Straub", email = "ben.x.straub@gsk.com", role = c("aut", "cre")),
+    person("Stefan", "Bundfuss", role = "aut"),
+    person("Thomas", "Neitmann", role = "aut"),
+    person("Samia", "Kabi", role = "aut"),
+    person("Gordon", "Miller", role = "aut"),
+    person("Ross", "Farrugia", role = "aut"),
+    person("Annie", "Yang", role = "aut"),
+    person("Robin", "Koeger", role = "aut"),
+    person("Pooja", "Kumari", role = "aut"),
+    person("Alice", "Ehmann", role = "ctb"),
+    person("Vignesh", "Thanikachalam", role = "ctb"),
+    person("Ondrej", "Slama", role = "ctb"),
+    person("Shimeng", "Huang", role = "ctb"),
+    person("F. Hoffmann-La Roche AG", role = c("cph", "fnd")),
+    person("GlaxoSmithKline LLC", role = c("cph", "fnd"))
```

Comment:
Are all of these people actually listed as `@author` on at least one function? If so, everyone should be `"aut"` and no one `"ctb"`. If no, please remove them from the list.

Reply:
I removed most of these folks.  Ondrej is the only one who is on the list who is no longer in admiral.  Since he wrote some of the original dev functions I was going to keep him as contributor.